orders/tests.py

```diff
@@ -52,3 +52,14 @@ def test_dog_owner_id_is_deleted_when_dog_owner_is_deleted(self, create_order):
     def test_daycare_id_is_deleted_when_daycare_is_deleted(self, create_order):
         DayCare.objects.get(id=1).delete()
         assert Order.objects.get(id=create_order.id).daycare_id is None
+
+    def test_get_all_day_cares_available_on_dates_function(self):
+        start_date = "2022-05-02"
+        end_date = "2022-08-02"
+        day_care = DayCare.create(username='testuser', email='test@email.com', password='test_password',
```

Comment:
didn't we add a bunch of fixtures to conftest so we won't need this anymore?

Reply:
see https://github.com/redhat-beyond/Hotails/pull/89/commits/c3ef4074998c115f486216beefc1b26d73cd4d97#r874035611 and let me know what you think (:

Reply:
either:
1. use the one in the contest and deal with the difference
2. add one with capacity=1  (my least favorite...)
3. use fixture-factory to create it dynamically.

Reply:
Fixed with option 1.